src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/GetAzureStorageUsage.cs

```diff
@@ -22,12 +22,11 @@
 
 namespace Microsoft.Azure.Commands.Management.Storage.StorageAccount
 {
-    [GenericBreakingChange("Parameter 'Location' will change from optional to mandatory in a future release.")]
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "StorageUsage"), OutputType(typeof(PSUsage))]
     public class GetAzureStorageUsageCommand : StorageAccountBaseCmdlet
     {
         [Parameter(
-            Mandatory = false,
+            Mandatory = true,
```

Comment:
Please update the help files for the two cmdlets affected by this PR.

Reply:
Thanks for the catch! Help is updated.